docs/src/examples/components/Alert/Variations/AlertExampleOOF.shorthand.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { Alert } from '@stardust-ui/react'
+
+const AlertExampleInfo = () => (
+  <Alert
+    variables={{ oof: true }}
```

Comment:
@codepretty I'm not sure you're aware about the Alerts prototype we have at http://localhost:8080/prototype-alerts

It'd be great to update it by:
- deleting code for `BannerAlert` component from `docs/src/prototypes/alerts/AnimatedBannerAlert.tsx` as it's already covered by work in your PR
- using `Alert` directly in `AnimatedBannerAlert` in the same file
- testing the prototypes works as expected

I can help with it and we can do it as follow-up, let me know what you prefer.

Reply:
oh, i didn't realize we had a prototype for it. do we still need the prototype if the scenarios are covered in the component examples?

Reply:
@codepretty it still works, you can try it out at http://localhost:8080/prototype-alerts 
I'd say that we don't really need it anymore; the only value it has it that it's integrated with our mocked compose box so you can see how it's "attaching" itself to it to achieve nice styling

Reply:
I updated the prototype - we can decide to remove later if we want